packages/harmony/src/foundations/corner-radius/cornerRadius.ts

```diff
@@ -0,0 +1,10 @@
+export const cornerRadius = {
+  xs: 2,
+  s: 4,
+  m: 8,
+  l: 12,
+  xl: 16,
+  '2xl': 32
+}
+
+export type CornerRadius = typeof cornerRadius
```

Comment:
Should each of these type declarations be the whole object or the options? looks like we need `keyof typeof cornerRadius` if we want a union of the keys

Reply:
yeah good q, i was using this type in creating the full theme object, if we need to use the keys at some point, suppose we could do `CorderRadiusTypes = keyof CornerRadius` ?

Reply:
Yes I just saw below. Having a type for the options that can be used in components and stuff would be nice. For the name, yea Types or Options or something like that

Reply:
sweet will also export!